endor/src/db/init_database.js

```diff
@@ -0,0 +1,32 @@
+import Sequelize from 'sequelize';
+
+import sequelize from './sequelize';
+
+// When syncing tables, this corresponds to the 'force' option.
+// force: true will drop the table if it already exists
+const overwriteExistingTables = true;
+
+/**
+ * Creates the 'users' table.
+ */
+function initUsers() {
+  const User = sequelize.define('user', {
```

Comment:
This is really just template code. I will get fleshed out further when I create a branch for issue #16